backend/src/Controller/Admin/StoreOwnerSubscription/AdminStoreSubscriptionController.php

```diff
@@ -99,4 +113,183 @@ public function deleteAllStoreFutureSubscriptionsByStoreOwnerId(int $storeOwnerI
+     *              @OA\Property(type="integer", property="storeProfileId"),
+     *              @OA\Property(type="integer", property="packageId"),
+     *              @OA\Property(type="string", property="note"),
+     *      )
+     *    )
+     *  )
+     *
+     *
+     * @OA\Response(
+     *      response=201,
+     *      description="Returns subscription",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code"),
+     *          @OA\Property(type="string", property="msg"),
+     *          @OA\Property(type="object", property="Data",
+     *             @OA\Property(type="integer", property="id")
+     *      )
+     *    )
+     *  )
+     * 
+     * or
+     *
+     * @OA\Response(
+     *      response="default",
+     *      description="Return error.",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code", description="9351 or 9157"),
+     *          @OA\Property(type="string", property="msg", description="package not exist or store owner profile not exist!"),
+     *        )
+     *     )
+     * 
+     * @Security(name="Bearer")
+     */
+    public function createSubscription(Request $request): JsonResponse
+    {
+        $data = json_decode($request->getContent(), true);
+
+        $request = $this->autoMapping->map(stdClass::class, AdminCreateStoreSubscriptionRequest::class, (object)$data);
+
+        $violations = $this->validator->validate($request);
+        if (\count($violations) > 0) {
+            $violationsString = (string) $violations;
+
+            return new JsonResponse($violationsString, Response::HTTP_OK);
+        }
+
+        $result = $this->adminStoreSubscriptionService->createSubscription($request);
+
+        if($result === StoreProfileConstant::STORE_NOT_FOUND){
+            return $this->response(MainErrorConstant::ERROR_MSG, self::STORE_OWNER_PROFILE_NOT_EXIST);
+        }
+
+        if(isset($result->packageState)){
+
+            return $this->response(MainErrorConstant::ERROR_MSG, self::PACKAGE_NOT_EXIST);
+        }
+
+        return $this->response($result, self::CREATE);
+    }
+
+    /**
+     * admin: extra Subscription For Day By Admin
+     * @Route("extrasubscriptionforday", name="extraSubscriptionForDayByAdmin", methods={"POST"})
+     * @IsGranted("ROLE_ADMIN")
+     * @param Request $request
+     * @return JsonResponse
+     *
+     * @OA\Tag(name="Subscription")
+     *
+     * @OA\Parameter(
+     *      name="token",
+     *      in="header",
+     *      description="token to be passed as a header",
+     *      required=true
+     * )
+     *
+     * @OA\RequestBody(
+     *      description="create subscription For Day",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="status_code"),
+     *          @OA\Property(type="string", property="msg"),
+     *          @OA\Property(type="object", property="Data",
```

Comment:
request body has no Data, msg, and status code. :)

Reply:
done